package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"devDependencies": {
```

Comment:
odd formatting

Reply:
autogenerated by lerna, it used tabs apparently. Fixed